openmetadata-service/src/main/java/org/openmetadata/service/search/indexes/ContainerIndex.java

```diff
@@ -77,6 +64,12 @@ public Map<String, Object> buildESDoc() {
         getFQNParts(
             container.getFullyQualifiedName(),
             suggest.stream().map(SearchSuggest::getInput).collect(Collectors.toList())));
+    if (container.getOwner() != null) {
+      doc.put("owner", getOwnerWithDisplayName(container.getOwner()));
```

Comment:
may be we should index nulls as well, then we can support if the docs do not have owner or not

Reply:
Will handle this in other PR need some discussion on this with @karanh37 for UI inputs had a call with him.